man/osg-incommon-cert-request.1

```diff
@@ -44,41 +46,63 @@ May be specified more than once for additional SANs.
 .TP
 .BR \-d, \-\-directory
 The directory to write the host certificate(s) and key(s).
+.TP
+.BR \-O, \-\-orgcode
+Specify alternative organization and department codes for the InCommon \\
+Certificate Service. Defaults are Fermilab\'s codes.
```

Comment:
The backslash before the single quote ends up with a question mark in the output, please delete it.

Reply:
Deleted.